### ISSUE TITLE
Bad focus order for collaborator menu

### ISSUE THREAD

**Issue by juliushaertl** [nextcloud/text#2733]
The collaborator menu is not accessible directly after opening as it is put at the very end of the DOM tree, so there user basically needs to tab through the rest of the page in order to reach it

![Screenshot 2022-07-20 at 08 31 50](https://user-images.githubusercontent.com/3404133/179912936-214cff5c-91f4-4759-aba8-e7ace13b94f9.png)
![Screenshot 2022-07-20 at 08 34 21](https://user-images.githubusercontent.com/3404133/179913120-6d9d39af-c0a0-4c59-ad76-34aa123070df.png)

**Comment by vinicius73** [comment]
Waiting for next release of `@nextcloud/vue`

https://github.com/nextcloud/nextcloud-vue/pull/2941

**Comment by juliushaertl** [comment]
Done with https://github.com/nextcloud/text/pull/2794
